tests/web_api/basic_case/test_variables.py

```diff
@@ -140,8 +140,8 @@ def test_dated_variable_formulas_dates():
 
 
 def test_dated_variable_formulas_content():
-    formula_code_2016 = "def formula_2016_12(person, period, parameters):\n    age_condition = person('age', period) >= parameters(period).general.age_of_majority\n    return age_condition * parameters(period).benefits.basic_income  # This '*' is a vectorial 'if'. See http://openfisca.org/doc/coding-the-legislation/30_case_disjunction.html#simple-multiplication\n"
-    formula_code_2015 = "def formula_2015_12(person, period, parameters):\n    age_condition = person('age', period) >= parameters(period).general.age_of_majority\n    salary_condition = person('salary', period) == 0\n    return age_condition * salary_condition * parameters(period).benefits.basic_income  # The '*' is also used as a vectorial 'and'. See http://openfisca.org/doc/coding-the-legislation/25_vectorial_computing.html#forbidden-operations-and-alternatives\n"
+    formula_code_2016 = "def formula_2016_12(person, period, parameters):\n    age_condition = person('age', period) >= parameters(period).general.age_of_majority\n    return age_condition * parameters(period).benefits.basic_income  # This '*' is a vectorial 'if'. See https://openfisca.org/doc/coding-the-legislation/30_case_disjunction.html#simple-multiplication\n"
+    formula_code_2015 = "def formula_2015_12(person, period, parameters):\n    age_condition = person('age', period) >= parameters(period).general.age_of_majority\n    salary_condition = person('salary', period) == 0\n    return age_condition * salary_condition * parameters(period).benefits.basic_income  # The '*' is also used as a vectorial 'and'. See https://openfisca.org/doc/coding-the-legislation/25_vectorial_computing.html#forbidden-operations-and-alternatives\n"
```

Comment:
If this fix is due to the release of `3.1.3` for the country template, then the minimum version of `openfisca_country_template` should be updated in `setup.py`